mobile-app/lib/ui/views/learn/superblock/superblock_view.dart

```diff
@@ -68,27 +83,44 @@ class SuperBlockView extends StatelessWidget {
           notification.disallowIndicator();
           return true;
         },
-        child: ListView.separated(
-          separatorBuilder: (context, int i) => const Divider(
-            height: 0,
-            color: Colors.transparent,
-          ),
-          shrinkWrap: true,
-          itemCount: superBlock.blocks!.length,
-          physics: const ClampingScrollPhysics(),
-          itemBuilder: (context, i) => Padding(
-            padding: model.getPaddingBeginAndEnd(
-              i,
-              superBlock.blocks![i].challenges.length,
-            ),
-            child: Column(
-              children: [
-                BlockTemplateView(
-                    block: superBlock.blocks![i],
-                    isOpen: superBlock.blocks!.length <= 3)
-              ],
-            ),
-          ),
+        child: CustomScrollView(
```

Comment:
If we're managing the block open using a state variable then there was no need of changing from `ListView` to `CustomScrollView`

Reply:
CustomScrollView supports Slivers, that supports ` addAutomaticKeepAlives: true,` which keeps the state.

Reply:
Thanks, I thought I needed the custom scrollview as the state was not being memorized. But that was before I decided to put the state at the parent level instead of the block level.